awx/ui/src/components/LaunchButton/LaunchButton.test.js

```diff
@@ -128,7 +125,7 @@ describe('LaunchButton', () => {
     WorkflowJobTemplatesAPI.launch.mockImplementation(async () => {
       // return asynchronously so isLaunching isn't set back to false in the
       // same tick
-      await sleep(10);
+      await new Promise((resolve) => setTimeout(resolve, 10));
```

Comment:
Unsure if this is still needed.

Reply:
The test passes locally without `new Promise` and `sleep`.  However, the test, which is supposed to check if the button is disabled, never asserts that `disabled === true`, so I'm not really sure what this is actually testing.

Reply:
Good find @AlexSCorey